### ISSUE TITLE
[BUG] Broken display with RCON

### ISSUE THREAD

**Issue by ambroiseRabier** [WeLoveSquad/squad-server-info-discord-bot#25]
First of all, thanks for your hard work on this project.

The bot recreate the message often, making it blink every 10sec or so.
![image](https://github.com/WeLoveSquad/squad-server-info-discord-bot/assets/16115874/85ebd2a3-64b2-46cc-a524-ccb8ef736a7a)

Disabling RCON make it work correctly, but without the bottom panel.

Relevant logs:

```
[2023-08-24T20:54:47.016Z] debug:   [InfoChannelHandler] Editing player info message [1144374206935605318]
DiscordAPIError[10008]: Unknown Message
    at handleErrors (/home/node/app/node_modules/@discordjs/rest/dist/index.js:640:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SequentialHandler.runRequest (/home/node/app/node_modules/@discordjs/rest/dist/index.js:1021:23)
    at async SequentialHandler.queueRequest (/home/node/app/node_modules/@discordjs/rest/dist/index.js:862:14)
    at async REST.request (/home/node/app/node_modules/@discordjs/rest/dist/index.js:1387:22)
    at async MessageManager.edit (/home/node/app/node_modules/discord.js/src/managers/MessageManager.js:176:15)
    at async InfoChannelHandler.updatePlayerInfoMessages (file:///home/node/app/build/handler/info-channel.handler.js:214:17)
    at async InfoChannelHandler.updateInfoMessages (file:///home/node/app/build/handler/info-channel.handler.js:152:17)
    at async InfoChannelHandler.syncServerInfos (file:///home/node/app/build/handler/info-channel.handler.js:142:13)
    at async Timeout._onTimeout (file:///home/node/app/build/handler/info-channel.handler.js:108:17) {
  requestBody: {
    files: [],
    json: {
      content: undefined,
      tts: false,
      nonce: undefined,
      embeds: [Array],
      components: [Array],
      username: undefined,
      avatar_url: undefined,
      allowed_mentions: undefined,
      flags: 0,
      message_reference: undefined,
      attachments: undefined,
      sticker_ids: undefined,
      thread_name: undefined
    }
  },
  rawError: { message: 'Unknown Message', code: 10008 },
  code: 10008,
  status: 404,
  method: 'PATCH',
  url: 'https://discord.com/api/v10/channels/1144299478527000606/messages/1144374206935605318'
}
```

**Comment by Mikerosoft25** [comment]
Hey, sorry for the late reply. I did not receive any notification about this issue, which is why I just noticed it now.

The error in your log can sometimes happen, if the request to the Discord API fails to edit the existing message. This is a problem on discord's end though. However in the latest version this should not crash the bot and simply result in sending a new message + automatically removing the old message in the future.

Do you remember which version you were using when the error occurred?
Could you try if the latest version (2.6.5) fixes the issue? I fixed some problems where the bot could not edit the existing message, resulting in multiple player-info messages in the same channel when the Discord API had some issues.

**Comment by Mikerosoft25** [comment]
@ambroiseRabier 
I have released version `2.7.0` that improves the logic of updating the embeds. Your issue should not occur in the new version, which is why i will close this issue. Feel free to create a new issue if the problem still occurs in the future.
